src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-kubernetes-proxy.mdx

```diff
@@ -9,87 +9,22 @@ redirects:
   - /docs/integrations/kubernetes-integration/installation/configure-kubernetes-proxy
 ---
 
-If you are running our Kubernetes integration with a proxy, you need to configure each component (the [infrastructure agent](/docs/infrastructure/install-infrastructure-agent), logging, Kubernetes events, etc.) to correctly work with your proxy.
+If you are running our Kubernetes integration with a proxy, you need to configure each component (the [infrastructure agent](/docs/infrastructure/install-infrastructure-agent), logging and synthetics minion) to correctly work with your proxy.
 
 ## Install the infrastructure agent with a proxy [#infrastructure-agent]
 
-There are three options to install the infrastructure agent with a proxy:
-
-* Installing with chart `newrelic-infrastructure`:
-  * The proxy can be configured setting the configuration option `config.proxy` as described in the [values.yaml](https://github.com/newrelic/helm-charts/blob/556d7d970d65ac1a9dcd30d6b7093b1e163cb281/charts/newrelic-infrastructure/values.yaml#L139) of the chart.
-    The whole `config` object is used to generate a configMap that is mounted as the file [/etc/newrelic-infra.yml](https://github.com/newrelic/helm-charts/blob/556d7d970d65ac1a9dcd30d6b7093b1e163cb281/charts/newrelic-infrastructure/templates/daemonset.yaml#L172) used to configure the agent.
-* Installing with chart `nri-bundle` as a dependency:
-
-  * The proxy can be configured setting the configuration option `newrelic-infrastructure.config.proxy`. The configuration option is passed [down to the dependency](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle#configure-dependencies) `newrelic-infrastructure` modifying the `values.yaml` of `nri-bundle`:
+The proxy can be configured setting the configuration option `global.proxy` and configure all the kubernetes integration at the same time. The global configuration is [shared with the dependencies](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle#configure-dependencies) by modifying the `values.yaml` of `nri-bundle`:
 
     ```yaml
-    newrelic-infrastructure.config.proxy: https://user:password@hostname:port
+    global.proxy: https://user:password@hostname:port
     ```
-* Installing with manifest:
-  * When installing [through the manifest](https://download.newrelic.com/infrastructure_agent/integrations/kubernetes/newrelic-infrastructure-k8s-latest.yaml), the proxy can be configured adding the `NRIA_PROXY` environment variable to the environment variable section of the DaemonSet configuration.
-  * The environment variable `NRIA_VERBOSE` in the DaemonSet YAML can be used as an example of the environment variable configuration.
+
+This will configure all the infrastructure agents, the kubernetes integration, the Metrics Adapter and the Prometheus OpenMetrics at the same time.
```

Comment:
also nri-kube-events and the operator?

Reply:
`nri-kube-events` use an agent that gets its config from the common library so it should be configures with the proxy automatically:
https://github.com/newrelic/nri-kube-events/blob/main/charts/nri-kube-events/templates/_helpers.tpl#L40-L45
https://github.com/newrelic/helm-charts/blob/master/library/common-library/templates/_agent-config.tpl#L13-L15

Reply:
About the operator... You are right, we do not support proxy in the operator yet.

Reply:
Solved in the commit [#5272112](https://github.com/newrelic/docs-website/pull/7322/commits/52721120b118b37ff91856396a750b4964b520c4) adding a callout that we do not support it.